sound/soc/sof/ipc4-topology.c

```diff
@@ -1065,9 +1065,14 @@ static int sof_ipc4_init_output_audio_fmt(struct snd_sof_dev *sdev,
 					  struct sof_ipc4_base_module_cfg *base_config,
 					  struct sof_ipc4_available_audio_format *available_fmt,
 					  u32 out_ref_rate, u32 out_ref_channels,
-					  u32 out_ref_valid_bits)
```

Comment:
commit message:
"When PCM1 is started" seems to be "When PCM**0** is started,"

Reply:
> no I meant when PCM1 is started @bardliao. WHen PCM0 is started, things will be fine because we're dealing with PIn 0 already. Its PCM1 that becomes probelmatic

@ranj063 I didn't get "When PCM1 is started, based on the current output format selection logic,
the output format for module-copier.N.1 will be set to S32_LE (based on
the pin 0's output format and the pipeline params will be modified to
reflect this."
Why the output format for module-copier.N.1 will be set based on the pin 0's output format? Pin 0 is not involved, right? The issue looks more like "how do we select out_ref_valid_bits" to me.